assets/dev/js/editor/regions/panel/pages/elements/views/category.js

```diff
@@ -1,3 +1,5 @@
+import { AppManager } from 'elementor-modules/promotions/assets/js/react/app-manager';
```

Comment:
I am not sure about the naming. It isn't clear which 'app' this is.

Possibly we could rename it to 'PromotionPopupManager', but I am open for suggestions.